tests/ks_initial/generate_expected_result.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+import argparse
+import sys
+import os
+from pathlib import Path
+from hadrons.functions import Jaffe_theory
+from testing_parameters import MATRIX_DF
+
+# absolute path of the file as string
+ABS_PATH = str(Path(__file__).parent.absolute())
+
+
+def calculate_expected_df():
+    result_df = pd.DataFrame()
+    for _, data in MATRIX_DF.iterrows():
+        Jaffe_df = Jaffe_theory(x=data.E_MeV_u, **data, input_is_LET=False)
+        result_df = pd.concat([result_df, Jaffe_df], ignore_index=True)
+
+    return result_df
+
+
+def configure_parser():
```

Comment:
what do you mean by
`overriding previous generation`
?

do you mean to generate new set of reference results ?
If yes, then such code should go to a different script, which may still be under `tests` directory, but not prefixed with `test_`

Reply:
Override previously generated file if present. Would "-f/--force"  be more clear?

Reply:
`previous generation` was not clean, in data science we are more used to `reference results` or `reference data`. 
The option is OK, the description needs clarification